### PR TITLE
Get macOS Builds from CI

### PR DESCRIPTION
**Pull request by IsaacMarovitz**

### CODE REVIEW

**Comment by gdkchan** [comment]
I don't think we should change this until all changes are upstreamed, or at least the build from CI is close to macos1 release, otherwise users will be confused when their games runs worse or not at all on the newer build.

**Comment by IsaacMarovitz** [comment]
> I don't think we should change this until all changes are upstreamed, or at least the build from CI is close to macos1 release, otherwise users will be confused when their games runs worse or not at all on the newer build.

I agree, I think once buffer mirrors is upstreamed this is probably safe to do. I'm gonna keep the PR around so that when the time comes it's an easy change.

File: src/views/DownloadPage.vue [hunk 1]
```diff
@@ -48,28 +47,17 @@ const fetchBuilds = async () => {
           windowBuildUrl.value = asset.browser_download_url;
         } else if (asset.name.endsWith("linux_x64.tar.gz")) {
           linuxBuildUrl.value = asset.browser_download_url;
+        } 
+      } else if (asset.name.startsWith("test-ava-ryujinx")) {
+        if (asset.name.endsWith(".app.tar.gz")) {
```
Comment (TSRBerry) [inline src/views/DownloadPage.vue:52]:
```suggestion
        if (asset.name.endsWith("macos_universal.app.tar.gz ")) {
```
Just for consistency with the other cases here.

**Review by TSRBerry — COMMENTED** [review]
Lgtm!

**Review by TSRBerry — APPROVED** [review]
Lgtm, thanks!

**Review by AcK77 — APPROVED** [review]

**Pull request merged**